<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>아세토코르사 컴패티치오네 서버매니저</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>42191111.png</normaloff>42191111.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_server">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>391</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string>서버 설정</string>
    </property>
    <widget class="QLabel" name="label_serverName">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>90</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>서버 이름</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_serverName">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>270</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_password">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>45</y>
       <width>90</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>서버 비밀번호</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_password">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>45</y>
       <width>270</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_adminPassword">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>90</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>관리자 비밀번호</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_adminPassword">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>70</y>
       <width>270</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_spectatorPassword">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>95</y>
       <width>90</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>관전자 비밀번호</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_spectatorPassword">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>95</y>
       <width>270</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_maxCarSlots">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>155</y>
       <width>90</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>최대 차량 슬롯</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_maxCarSlots">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>155</y>
       <width>42</width>
       <height>20</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_trackMedalsRequirement">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>180</y>
       <width>42</width>
       <height>20</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_trackMedalsRequirement">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>트랙 메달 요구량</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_safetyRatingRequirement">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>205</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>SA 요구량</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_safetyRatingRequirement">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>205</y>
       <width>42</width>
       <height>20</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_isRaceLocked">
     <property name="geometry">
      <rect>
       <x>235</x>
       <y>155</y>
       <width>90</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>레이스 난입 금지</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_isRaceLocked">
     <property name="geometry">
      <rect>
       <x>367</x>
       <y>155</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_dumpLeaderboards">
     <property name="geometry">
      <rect>
       <x>235</x>
       <y>130</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>결과 순위표 저장</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_dumpLeaderboards">
     <property name="geometry">
      <rect>
       <x>367</x>
       <y>130</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_allowAutoDQ">
     <property name="geometry">
      <rect>
       <x>235</x>
       <y>180</y>
       <width>90</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>자동 실격 처리</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_allowAutoDQ">
     <property name="geometry">
      <rect>
       <x>367</x>
       <y>180</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_maxConnections">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>130</y>
       <width>42</width>
       <height>20</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_maxConnections">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>90</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>최대 연결 수</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_event">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>10</y>
      <width>381</width>
      <height>581</height>
     </rect>
    </property>
    <property name="title">
     <string>이벤트 설정</string>
    </property>
    <widget class="QLabel" name="label_track">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>30</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>트랙</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_track">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>321</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_track_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>45</y>
       <width>30</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>차량</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_track_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>45</y>
       <width>321</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_time">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>361</width>
       <height>51</height>
      </rect>
     </property>
     <property name="title">
      <string>시간</string>
     </property>
     <widget class="QLabel" name="label_preRaceWaitingTimeSeconds">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>100</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>레이스 준비시간</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_sessionOverTimeSeconds">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>20</y>
        <width>100</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>레이스 추가시간</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_sessionOverTimeSeconds">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>20</y>
        <width>42</width>
        <height>20</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_preRaceWaitingTimeSeconds">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>20</y>
        <width>42</width>
        <height>20</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_weather">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>361</width>
       <height>141</height>
      </rect>
     </property>
     <property name="title">
      <string>기상</string>
     </property>
     <widget class="QLabel" name="label_ambientTemp">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>100</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>외부 온도</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdNumber_ambientTemp">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>20</y>
        <width>64</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QSlider" name="horizontalSlider_ambientTemp">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>20</y>
        <width>181</width>
        <height>22</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdNumber_cloudLevel">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>50</y>
        <width>64</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QSlider" name="horizontalSlider_cloudLevel">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>50</y>
        <width>181</width>
        <height>22</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_cloudLevel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>100</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>구름이 낀 정도 </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_rain">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>100</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>강수량 </string>
      </property>
     </widget>
     <widget class="QSlider" name="horizontalSlider_rain">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>80</y>
        <width>181</width>
        <height>22</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdNumber_rain">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>80</y>
        <width>64</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QSlider" name="horizontalSlider_weatherRandomness">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>110</y>
        <width>181</width>
        <height>22</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_weatherRandomness">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>100</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>날씨 변동성</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdNumber_weatherRandomness">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>110</y>
        <width>64</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_session">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>280</y>
       <width>361</width>
       <height>281</height>
      </rect>
     </property>
     <property name="title">
      <string>세션</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_start">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>510</y>
      <width>391</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>서버 시작</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_rule">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>391</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string>규칙 설정</string>
    </property>
    <widget class="QGroupBox" name="groupBox_pitstop">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>171</width>
       <height>221</height>
      </rect>
     </property>
     <property name="title">
      <string>피트스톱</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_pitstop_2">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>191</width>
       <height>221</height>
      </rect>
     </property>
     <property name="title">
      <string>어시스트</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
